content/developer/composer/customization.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: bt_wiki
+title: Customization
+category: Composer
+draft: false
+weight: 600
+userdata_path: "/opt/cloudify-composer/backend/dev"
+config_path: "/opt/cloudify-composer/backend/conf"
+---
+
+{{< param cfy_composer_name >}} application provides basic support for customizing the layout.
+Main features are:
+
+* adding custom logo and setting main application color
+* styling the HTML elements
+* changing text labels 
+
+When introducing changes described in this page you should be aware that: 
+
+* to see the applied changes you need to restart {{< param cfy_composer_name >}} 
+service on the {{< param cfy_manager_name >}} machine by executing: 
+```shell script
+sudo service cloudify-composer.service restart
+```
+* in {{< param cfy_manager_name >}} Cluster environment, 
+these changes should be applied to every {{< param cfy_manager_name >}} in the cluster
+
+
+## Theme setup
```

Comment:
Order in which information is given in this section is counter intuitive. What's in lines 49-53 is vital and should be placed immediately after line 35. It should be also explicitly stated what should be placed in the created file - the sentence 'Default configuration file is available at ...' doesn't really tell how created file is related to this default configuration file (overriding).
Example is least important and should be moved to the end.

Reply:
Changed it a bit.